tests/chainer_tests/test_link.py

```diff
@@ -2466,4 +2649,32 @@ def test_to_intel64(self):
         assert l.child.to_method_called == 1
 
 
+@_inject_backend_tests
```

Comment:
This test did not exist in the previous PR.

Reply:
Ok!